node/npm/glide/tsconfig.json

```diff
@@ -0,0 +1,30 @@
+{
+    "compilerOptions": {
+        "target": "esnext" /* Set the JavaScript language version for emitted JavaScript and include compatible library declarations. */,
+        "allowJs": true,
+        "esModuleInterop": true,
+        "baseUrl": "./",
+        "outDir": "./build-ts/mjs" /* Specify an output folder for all emitted files. Ovrride in tsconfig-cjs */,
```

Comment:
why mjs?

Reply:
shouldn't this field be empty?

Reply:
When leaving those fields empty you get errors in your IDE if you use some extensions since they check for tsconfig configuration.
We override those value anyway, but leaving default values helping the extensions to find what conf to follow.

Reply:
What i can do to improve the clarity of it is adding it under separate section with explainetion comment as the header